dev-libs/expat/patches/expat-2.6.2.patchset

```diff
@@ -33,19 +33,19 @@ index 0000000..c1aad9d
 +get_filename_component(_IMPORT_PREFIX "${_IMPORT_PREFIX}" PATH)
 +
 +set_target_properties(expat::expat PROPERTIES
-+  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/develop/headers${CMAKE_HAIKU_SECONDARY_ARCH_SUBDIR}"
++  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/system/develop/headers${CMAKE_HAIKU_SECONDARY_ARCH_SUBDIR}"
 +  INTERFACE_LINK_LIBRARIES "m"
 +)
 +
 +# Import target "expat::expat" for configuration "NoConfig"
 +set_property(TARGET expat::expat APPEND PROPERTY IMPORTED_CONFIGURATIONS NOCONFIG)
 +set_target_properties(expat::expat PROPERTIES
-+  IMPORTED_LOCATION_NOCONFIG "${_IMPORT_PREFIX}/lib/@LIBDIR_BASENAME@/libexpat.so.@SO_MAJOR@.@SO_MINOR@.@SO_PATCH@"
++  IMPORTED_LOCATION_NOCONFIG "${_IMPORT_PREFIX}/system/@LIBDIR_BASENAME@/libexpat.so.@SO_MAJOR@.@SO_MINOR@.@SO_PATCH@"
```

Comment:
Haven't checked this on 32bit (yet), so not sure if `@LIBDIR_BASENAME@` results in the secondary arch subdir path.

Reply:
Needs some investigation, I know we already do some sed magic in a few recipe related to this.

Reply:
Looks to be working OK on 32bit without this one. <me puzzled>